test_runner/performance/test_compute_startup.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import annotations
+
+import pytest
+import requests
+from fixtures.benchmark_fixture import MetricReport, NeonBenchmarker
+from fixtures.neon_fixtures import NeonEnvBuilder, PgBin
```

Comment:
I think you can move these imports behind an `if TYPE_CHECKING:` block

Reply:
Hm, do not see that we do this consistently in other tests. What is the benefit?

Reply:
Yeah it's hit or miss. When I can, I try to fix old code, and when I write new code, I follow the pattern. It can make execution times faster because you only import types during type checking passes (mypy for example).

Reply:
@bayandin, should we automate some check for this (if it's possible)? ^ Because, tbh, if something is not enforced with tools, there is a low chance it will remain sane for a long time

Reply:
It seems enabling https://docs.astral.sh/ruff/rules/#flake8-type-checking-tc should be enough. It can be done in a separate PR